packages/trader/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -0,0 +1,65 @@
+import React         from 'react';
+import PropTypes     from 'prop-types';
+import FullPageModal from 'App/Components/Elements/FullPageModal/full-page-modal.jsx';
+import Button        from 'App/Components/Form/button.jsx';
+import Dropdown      from 'App/Components/Form/DropDown';
+// import InputField    from 'App/Components/Form/InputField/input-field.jsx';
+import { connect }   from 'Stores/connect';
+
+// const onClose = (ui) => {
+//     ui.toggleUnsupportedContractModal(false);
+// };
+
+const AccountSignup = ({ onSignup, residence_list }) => {
+    const userInput = { password: null, residence: null };
+    const list      = residence_list;
```

Comment:
isn't `list` a bit general and vague?

Reply:
@negarn It's temporary, that component still has a good amount of clean-up. I will leave comments as yellow tape.